nsis/src/nsis/file/NsisExecutable.java

```diff
@@ -29,28 +34,54 @@ public static NsisExecutable createNsisExecutable(GenericFactory factory,
 	private void initNsisExecutable(GenericFactory factory, ByteProvider bp,
 			SectionLayout layout) throws IOException {
 		this.reader = new FactoryBundledWithBinaryReader(factory, bp, true);
```

Comment:
I understand this like isn't touched by this PR, but consider explicitly explaining what `true` argument does in this line:
`new FactoryBundledWithBinaryReader(factory, bp, /*isLittleEndian=*/ true);`

Reply:
should I make it a constant or just explain it with a comment?

Reply:
Just a comment is fine

Reply:
Added where applicable